package.json

```diff
@@ -112,8 +113,7 @@
     "passport-oauth2": "^1.6.1",
     "redis": "^3.1.2",
     "reflect-metadata": "^0.1.13",
-    "superagent": "^6.1.0",
```

Comment:
So your removing universal-analytcis and replacing it with ga-gtag? This might be useful for the ticket? https://dsdmoj.atlassian.net/browse/NN-3878 if so worth putting a comment in it?